OpenMage/magento-lts
# Removed jQuery dependencies from swatches in RWD theme

### Description (*)


With this PR, it is possible to extend RWD Theme by inheriting the swatches functionality without the need to unnecessarily load jQuery. 

### Fixed Issues (if relevant)

1. Fixes OpenMage/magento-lts#<issue_number>

### Manual testing scenarios (*)

1. Comment out jQuery lib in the page.xml file.
```xml
            <block type="page/html_head" name="head" as="head">
                <action method="addJs"><script>prototype/prototype.js</script></action>
                
```
2. Test swatches in product list and PDP. 
In both cases image gallery must swap with selected option, in PDP price must change if there is a supplement.

PS: It should work in both cases, with and without jQuery.
It is normal to have a lot of console errors related to jQuery used in other JavaScript.

### Questions or comments

It has been very helpful for creating lightweight themes. In the future, if we can also eliminate PrototypeJs from the libraries used for configurable products as well, it would be awesome.

### Contribution checklist (*)
